app/assets/javascripts/float-kit/addon/lib/d-toast-instance.js

```diff
@@ -13,7 +13,11 @@ export default class DToastInstance {
 
   constructor(owner, options = {}) {
     setOwner(this, owner);
-    this.options = { ...TOAST.options, ...options };
+    this.options = {
+      ...TOAST.options,
+      ...{ component: DDefaultToast },
```

Comment:
This one also appeared in a retained trace, Im not sure important a component and storing it in an array was a great idea. It was probably not causing much harm though.